src/index.js

```diff
@@ -1,33 +1,230 @@
+// import { cLearALL, handleCheckboxchange } from './Modules/checkbox.js';
+// import DeleteTask from './Modules/Delete.js';
```

Comment:
- [ ] As pointed out earlier, you should remove all commented code in all your codebase but there are still some comments